Ubers/250_Ho-Oh.yaml

```diff
@@ -40,7 +40,7 @@ tags: [ubers]
 item: [Life Orb, Power Herb, Meadow Plate, White Herb]
 ability: [Pressure]
 moves:
-    - [Overheat, Fire Blast]
+    - [Overheat, Fire Blast, Fire Spin]
```

Comment:
Fire spin is a LOT weaker than the other 2 options, remember that gen 4 has neither the base power or passive damage buffs, and not fitting for ubers

Reply:
dang, aight